typescript/type-converter.js

```diff
@@ -59,8 +59,10 @@ const getName = (node, src) => {
  * @returns {string} modified jsDoc comment with appended @param tags
  * 
  */
-const convertParams = (jsDoc = '', node, src, parentName = null) => {
-  node.type.parameters.forEach(parameter => {
+const convertParams = (jsDoc = '', node, src) => {
+  const parameters = node.type && node.type.parameters || node.parameters
```

Comment:
Optional: `node.type?.parameters`

Reply:
Changed, but current eslint doesn't support optional chaining operator.
Will be checked after the beta version because it changes linting rules too.

![Screenshot from 2022-01-19 14-03-54](https://user-images.githubusercontent.com/85543813/150137165-768c37c9-4220-448d-9857-fd511e25d23c.png)

Reply:
👍